priv/templates/nimble.phx.gen.template/Dockerfile.eex

```diff
@@ -0,0 +1,49 @@
+FROM hexpm/elixir:1.11.0-erlang-23.1.1-alpine-3.12.0 AS build
+
+# install build dependencies
+RUN apk add --no-cache build-base npm git && \
+    mix local.hex --force && \
+    mix local.rebar --force
+
+# prepare build dir
+WORKDIR /app
+
+# set build ENV
+ENV MIX_ENV=prod
+
+# install mix dependencies
+COPY mix.exs mix.lock ./
+COPY config config
+RUN mix do deps.get, deps.compile
+
+# build assets
+COPY assets/package.json assets/package-lock.json ./assets/
+RUN npm --prefix ./assets ci --progress=false --no-audit --loglevel=error
+
+COPY priv priv
+COPY assets assets
+RUN npm run --prefix ./assets deploy
+RUN mix phx.digest
+
+# compile and build release
+COPY lib lib
+# uncomment COPY if rel/ exists
+# COPY rel rel
+RUN mix do compile, release
+
+# prepare release image
+FROM alpine:3.12.0 AS app
+RUN apk add --no-cache openssl ncurses-libs
+
+WORKDIR /app
+
+RUN chown nobody:nobody /app
+
+USER nobody:nobody
```

Comment:
Not sure if `nobody` is a standard non-root user name and group but maybe having something more descriptive would be better

Reply:
changed it to `app_group` and `app_user` @olivierobert